client/flutter/lib/pages/main_pages/app_tab_router.dart

```diff
@@ -27,9 +29,9 @@ class AppTabRouter extends StatelessWidget {
           case 1:
             return LatestNumbersPage();
           case 2:
-            return LearnPage();
+            return ProtectYourself(dataSource: FactContent.protectYourself);
           case 3:
-            return CheckUpPage();
+            return TravelAdvice(dataSource: AdviceContent.travelAdvice);
```

Comment:
Do we need to do this? I think it's ok to leave the placeholders, we'll be filling them in shortly anyway...